src/common/intl/index.js

```diff
@@ -63,6 +63,7 @@ export class IntlProvider extends Component {
   render () {
     const { lang, children } = this.props
     return <IntlProvider_
+      key={lang}
```

Comment:
This really looks like an ugly hack.

The least you can do is add a link to the page where it is suggested.